server/helpers/db.js

```diff
@@ -33,6 +33,7 @@ module.exports = async () => {
   );
 
   // Cache the database connection and return the connection
+  // eslint-disable-next-line require-atomic-updates
```

Comment:
if there's time refactor this so we don't get this lint error

Reply:
linter might be tweaking on this one

Reply:
It's from the way that value is being updated. check [this](https://eslint.org/docs/rules/require-atomic-updates)

Reply:
i followed [this guide](https://zeit.co/guides/deploying-a-mongodb-powered-api-with-node-and-now)